Examples/test-suite/li_attribute_template.i

```diff
@@ -2,6 +2,11 @@
 
 %include <exception.i>
 
+// Fortran doesn't have native attribute wrap support
+#ifdef SWIGFORTRAN
+#pragma SWIG nowarn=507
```

Comment:
To avoid masking a problem, please instead put this test in the set of broken tests like li_attribute.

Reply:
I meant that the language natively doesn't have "attributes" in the way that Python does, so I added an `attributes.i` file that simply warned about the non-native feature. I'll remove that file and the warning code.